first session/assignment/shoppingmall.css

```
@@ -0,0 +1,126 @@
+body {
+    font-family: Arial, sans-serif;
+}
+
+.hamburger-menu .menu {
+    display: none;
+}
+
+
+.hamburger-icon {
+    background: none;
+    border: none;
+    font-size: 24px;
+    cursor: pointer;
+    padding: 10px;
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    padding: 10px;
+    position: fixed;
+    width: 100%;
+    top: 0;
+}
```

지금 header의 background 색상을 지정해주지 않아서 기본값인 transparent(투명) 색이 되었어요. 그래서 스크롤을 내리면 뒤에 있는 사진들이 header 뒤로 비쳐보여서 UX가 굉장히 별로인 상황이죠,,,,,
이를 방지하기 위해 header의 background를 white로 지정해주면 밑에 있는 사진들이나 다른 요소들이 뒤에 비쳐보이지 않는 효과를 얻을 수 있답니다 :)
```suggestion
header{
    display: flex;
    justify-content: space-between;
    padding: 10px;
    position: fixed;
    width: 100%;
    top: 0;
    
    background-color: white;
}
```